src/main/java/com/crud/tasks/service/MailService.java

```diff
@@ -22,13 +23,14 @@ public void sendQtyEmail(long tasksCount) {
         simpleEmailService.send(new Mail(
                 adminConfig.getAdminMail(),
                 QTY_EMAIL_SUBJECT,
-                mailCreatorService.buildTasksQtyEmail("Currently in database you got: " + tasksCount + (tasksCount == 1 ? " task" : " tasks"))));
+                mailCreatorService.buildTasksQtyEmail("" + tasksCount)));
```

Comment:
Skoro treść maila zależy tylko od ilości zadań, to powinnaś przekazywać tę ilość jako typ liczbowy, nie `String`. Jeśli faktycznie jest potrzeba rzutowania na `String`, zrób to wewnątrz `MailCreatorService`. Jednak wątpię, większość mechanizmów szablonów bez problemu przyjmie typy liczbowe w kontekście, sprawdź czy tak jest dla ThymeLeaf.